Code/Framework/AzCore/AzCore/RTTI/BehaviorContext.h

```diff
@@ -69,8 +69,11 @@ namespace AZ
     };
 
     /**
-     * Stores information about a function parameter (no instance). During calls we use \ref BehaviorValueParameter which in addition
-     * offers value storage and functions to interact with the data
+     * Reflects information about a C++ function parameter, mostly for use in BehaviorContext Methods and Events. It only provides a
```

Comment:
I don't think this comment format is doxygen? so should we have documentation updated as well?